src/hooks/dx9.rs

```diff
@@ -123,8 +125,8 @@ unsafe extern "system" fn imgui_wnd_proc(
     WPARAM(wparam): WPARAM,
     LPARAM(lparam): LPARAM,
 ) -> LRESULT {
-    if IMGUI_RENDERER.is_some() {
-        match IMGUI_RENDERER.as_mut().unwrap().try_lock() {
+    if IMGUI_RENDERER.get().is_some() {
+        match IMGUI_RENDERER.get_mut().unwrap().try_lock() {
```

Comment:
Small nit, just noticed this now; even though it's not pertinent to this PR, while we're at it I'd lighten up the syntax by compressing the if and match in the same statement:
```suggestion
        if let Some(Ok(renderer)) = IMGUI_RENDERER.get_mut().map(|m| m.try_lock()) {
```

Reply:
Done. However, I had to use Some(Some()) instead, a bit ugly but eh, it was ugly anyways 😄

Reply:
Oh, in that case, my bad, thought it was an `Option<Result>`. In those cases, `and_then` in place of `map` has you covered 😅

Reply:
I'm not too sure of how to write it as my idiomatic Rust skills are not quite there yet.. 😄 

I'll leave it as is for now.